app/assets/stylesheets/automated_tests.scss

```diff
@@ -80,11 +65,34 @@ div#autotest-manager {
           font-size: 1em;
         }
       }
+
+      div {
+        margin: 0;
+      }
     }
-    div.btn-group {
-      max-width: 50%;
+
+    .array-item-list > .array-item {
+      border: 1px solid grey;
+      border-radius: $radius;
+      display: flex;
+      padding: 10px;
+
+      > .col-xs-9 {
+        flex: 0.8;
+      }
+
+      > .col-xs-3 {
+        flex: 0.2;
+      }
```

Comment:
Could we also add 

```
      + .array-item {
        margin-top: 10px;
      }
```
to make sure there is a bit of spacing between the array items? Right now they're squished together which makes the borders run into each other a bit.